### PR TITLE
Making use of QueueingThreadPoolExecutor 

### PR DESCRIPTION
**Pull request by tenevo**
Hello @kaikreuzer , @JochenHiller 
here is the PR regarding this Issue [ThreadPool rejected execution problem/excessive logging](https://github.com/jupnp/jupnp/issues/8) and some minor convenience changes 
Thanks

### CODE REVIEW

File: bundles/org.jupnp/pom.xml [hunk 1]
```diff
@@ -16,5 +16,13 @@
 	<name>jUPnP Core Library</name>
 
 	<packaging>eclipse-plugin</packaging>
-
+	
+	<dependencies>
```
Comment (kaikreuzer) [inline bundles/org.jupnp/pom.xml]: How is this related to the rest of your PR?

Comment (tenevo) [inline bundles/org.jupnp/pom.xml]: It was the completion of a missing dependency to start the bundles/org.jupnp/src/main/java/org/jupnp/Main.java

Comment (kaikreuzer) [inline bundles/org.jupnp/pom.xml]: Not sure I understand this. The bundles are built through Tycho and thus use the [jupnp.target](https://github.com/tenevo/jupnp/blob/b73837a0e3eeb98e2047481b38539b7044f95c5b/targetplatform/jupnp.target#L5-L6) to compile against - and this correctly includes the apache http client.

Comment (tenevo) [inline bundles/org.jupnp/pom.xml]: The thing is... I was not able to start (run) this Main class from my Eclipse IDE, due to this missing dependency.
Adding the dependency in the pom solves the problem.

Comment (tenevo) [inline bundles/org.jupnp/pom.xml]: Is it possible to sign previously commited changes? if yes, tell me please how.
Or do you mean it for future commits? Since the description [here](https://github.com/jupnp/jupnp/blob/master/CONTRIBUTING.md#sign-your-work) is only for "current" commits.
Thanks!

Comment (sjka) [inline bundles/org.jupnp/pom.xml]:
```
git rebase -i <predecessor-commit-id-of-your-commits>
```

then mark every commit you need to change with "reword" instead of "pick", save and exit. Git will then let you fix each commit's message.

You will have to `git push -f <origin> <branch>` then (i.e. force) in order to override the existing branch on the server.

Comment (kaikreuzer) [inline bundles/org.jupnp/pom.xml]: > Adding the dependency in the pom solves the problem.

If the Maven build works without it, please remove it again.

Any luck with squashing and signing your commits?

**Comment by kaikreuzer** [comment]
Closing due to inactivity.
